### DISCUSSION TITLE
what's the diffrence between the suffix version of k3s1 and k3s2？i can also see the suffix version of k3s3

### DISCUSSION THREAD

**Discussion by wang-xiaowu** [k3s-io/k3s#9034] [Q&A] [answered]
what's the diffrence between the suffix version of k3s1 and k3s2？i can also see the suffix version of k3s3

**Comment by brandond** [comment, marked as answer]
We increment the release tag when we need to do another release of the same upstream patch release. In this case, we ran into issues with our CI infrastructure that necessitated several rounds of releases until we were able to address all the issues. There are no meaningful differences between the releases.
